feat: beautiful code markdown

juliahermak opened the pull request:
Before:
![Screenshot 2023-10-09 22 15 51](https://github.com/Adamant-im/adamant-im/assets/122323015/9c555498-cf26-41e2-adce-634d1fb8d5b7)
After:
![Screenshot 2023-10-09 22 16 04](https://github.com/Adamant-im/adamant-im/assets/122323015/fd898087-5dc4-4666-b673-d98b60eaca6b)

bludnic commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Feat: Beautiful code markdown](https://trello.com/c/b5jkzwzx/381-feat-beautiful-code-markdown)

bludnic commented on src/assets/styles/components/_chat.scss:
```diff
@@ -537,10 +534,22 @@ $scroll-bar-width: 4px;
       }
+
+      code{
+        color: #ffffff;
```
```suggestion
        color: map-get($shades, 'white');
```

bludnic commented on src/assets/styles/components/_chat.scss:
```diff
@@ -537,10 +534,22 @@ $scroll-bar-width: 4px;
+        
         color: inherit;
         border-color: map-get($grey, 'darken-2');
+        background-color: map-get($adm-colors, 'secondary2-slightly-transparent') !important;
```
will it work without `!important`?

juliahermak commented:
Before:
![Screenshot 2023-10-11 04 23 29](https://github.com/Adamant-im/adamant-im/assets/122323015/5a1da63e-877c-41d9-bc54-d672e806f718)
After:
![Screenshot 2023-10-11 04 23 59](https://github.com/Adamant-im/adamant-im/assets/122323015/f4e71b46-4186-438b-938a-97938aadc082)

bludnic commented on src/assets/styles/components/_chat.scss:
```diff
@@ -236,15 +231,12 @@ $scroll-bar-width: 4px;
-      code:before {
-        content: '';
+      
+      code{
```
```suggestion
      code {
```

run prettier

juliahermak commented:
I added all the fonts from this list and added one more at Bludnic request.
![Screenshot 2023-10-15 11 07 18](https://github.com/Adamant-im/adamant-im/assets/122323015/7d4798be-7ecd-4c75-a52a-55463faf460f)

bludnic commented:
@martiliones see discussion in trello card. This select was added for testing purposes.
